libs/build_condaenv.sh

```diff
@@ -89,5 +93,5 @@ set -x
 set -u
 
 # generate modulefile from template
-$MODULES && update_modules python $name $version $python_version
+# $MODULES && update_modules python $name $version $python_version
```

Comment:
Why is module generation disabled?

Reply:
you could easily replace 
`conda activate regional_workflow`
with
`module load regional_workflow`.

When using the python environment, the module would still be `module load regional_workflow`.  
So you see, it matters not whether you build `regional_workflow` via conda or python virtual environment for the user, the user only needs to load the module for `regional_workflow`

The `module load regional_workflow` will activate and `module unload regional_workflow` will deactivate.
The user does not need to worry about native conda or venv calls.

Reply:
I will also note that the `module load` is the supported mechanism and not `conda activate` or `source $VENV/bin/activate`. 
This allows one to not know the inner details of how the module was built.